src/types/spot.ts

```diff
@@ -1929,3 +1924,3114 @@ export interface BSwapLiquidity {
+    scheduledData: number;
+    createTime: number;
+    status: string;
+    tranId: number;
+  }[];
+  count: number;
+}
+
+export interface ManagedSubAccountFuturesAssetsResponse {
+  code: string;
+  message: string;
+  snapshotVos: {
+    type: string;
+    updateTime: number;
+    data: {
+      assets: {
+        asset: string;
+        marginBalance: number;
+        walletBalance: number;
+      }[];
+      position: {
+        symbol: string;
+        entryPrice: number;
+        markPrice: number;
+        positionAmt: number;
+      }[];
+    };
+  }[];
+}
+
+export interface ManagedSubAccountMarginAssetsResponse {
+  marginLevel: string;
+  totalAssetOfBtc: string;
+  totalLiabilityOfBtc: string;
+  totalNetAssetOfBtc: string;
+  userAssets: {
+    asset: string;
+    borrowed: string;
+    free: string;
+    interest: string;
+    locked: string;
+    netAsset: string;
+  }[];
+}
+
+export interface SubAccountAssetsMasterResponse {
+  balances: {
+    asset: string;
+    free: string;
+    locked: string;
+  }[];
+}
+
+export interface ManagedSubAccountListParams {
+  email?: string;
+  page?: number;
+  limit?: number;
+}
+
+export interface ManagedSubAccountListResponse {
+  total: number;
+  managerSubUserInfoVoList: {
+    rootUserId: number;
+    managersubUserId: number;
+    bindParentUserId: number;
+    email: string;
+    insertTimeStamp: number;
+    bindParentEmail: string;
+    isSubUserEnabled: boolean;
+    isUserActive: boolean;
+    isMarginEnabled: boolean;
+    isFutureEnabled: boolean;
+    isSignedLVTRiskAgreement: boolean;
+  }[];
+}
+
+export interface SubAccountTransactionStatisticsResponse {
+  recent30BtcTotal: string;
+  recent30BtcFuturesTotal: string;
+  recent30BtcMarginTotal: string;
+  recent30BusdTotal: string;
+  recent30BusdFuturesTotal: string;
+  recent30BusdMarginTotal: string;
+  tradeInfoVos: {
+    userId: number;
+    btc: number;
+    btcFutures: number;
+    btcMargin: number;
+    busd: number;
+    busdFutures: number;
+    busdMargin: number;
+    date: number;
+  }[];
+}
+
+export interface ManagedSubAccountDepositAddressParams {
+  email: string;
+  coin: string;
+  network?: string;
+}
+
+export interface ManagedSubAccountDepositAddressResponse {
+  coin: string;
+  address: string;
+  tag: string;
+  url: string;
+}
+
+export interface EnableOptionsForSubAccountResponse {
+  email: string;
+  isEOptionsEnabled: boolean;
+}
+
+export interface ManagedSubAccountTransferTTLogParams {
+  startTime: number;
+  endTime: number;
+  page: number;
+  limit: number;
+  transfers?: string;
+  transferFunctionAccountType?: string;
+}
+
+export interface ManagedSubAccountTransferTTLogResponse {
+  managerSubTransferHistoryVos: {
+    fromEmail: string;
+    fromAccountType: string;
+    toEmail: string;
+    toAccountType: string;
+    asset: string;
+    amount: string;
+    scheduledData: number;
+    createTime: number;
+    status: string;
+    tranId: number;
+  }[];
+  count: number;
+}
```

Comment:
Same on moving the top level "wrapper" to the fn using this, so this interface is just the element in the array.